backend/open_webui/models/auths.py

```
@@ -143,32 +143,45 @@ def authenticate_user(self, email: str, password: str) -> Optional[UserModel]:
         except Exception:
             return None
 
-    def authenticate_user_by_api_key(self, api_key: str) -> Optional[UserModel]:
-        log.info(f"authenticate_user_by_api_key: {api_key}")
+    def authenticate_user_by_api_key(
+        self, api_key: str, db: Optional[Session] = None
+    ) -> Optional[UserModel]:
+        log.info(f"authenticate_user_by_api_key")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)